{{cookiecutter.project_slug}}/k8s/base/ingress.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: traefik.containo.us/v1alpha1
+kind: IngressRoute
+metadata:
+  name: frontend-ingress
+  annotations:
+    cert-manager.io/issuer: letsencrypt-prod
```

Comment:
Let's use `letencrypt-staging` by default to ensure the certificate quota isn't used up on misconfiguration.